packages/puppeteer-core/package.json

```diff
@@ -141,18 +142,19 @@
       "command": "tsc -b",
       "clean": "if-file-deleted",
       "dependencies": [
-        "clean:third_party",
         "generate:sources"
       ],
       "files": [
-        "src/**",
+        "**/tsconfig.*.json",
+        "**/tsconfig.json",
         "compat/**",
-        "third_party/**",
-        "**/tsconfig.*.json"
+        "src/**",
+        "third_party/**"
       ],
       "output": [
-        "lib/esm/**",
-        "lib/cjs/**"
+        ".wireit/third_party/**",
+        "lib/cjs/**",
+        "lib/esm/**"
```

Comment:
I think we should add `.tsbuildinfo` to files, but that can be done in another PR.

Reply:
It's unnecessary since changes to the compiled files are what matters. Output should always be about the artifacts using in the later steps (and caching). Anything else is just meta.